lib/tasks/events.rake

```diff
@@ -0,0 +1,187 @@
+require "date"
+
+namespace :events do
+  require_relative "events_fixtures"
+  users = EventsFixtures::USERS
+  asset_types = EventsFixtures::ASSET_TYPES
+  context_id = EventsFixtures::CONTEXT_ID
```

Comment:
You can pass arguments to rake tasks. For things like the context id it might be worth doing that, with a default or something.